src/emulator/bridge/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_library(
```

Comment:
This module has confusing naming. I don't agree with the name "bridge"

Reply:
Ended up removing and merging `bridge` with `host`. It is now much cleaner.